spec/rails_helper.rb

```diff
@@ -75,3 +75,5 @@
     with.library :rails
   end
 end
+
+require 'support/factory_bot'
```

Comment:
What is this for?

Reply:
Oh I see what this is for. Hmm I just pushed this to master though. I think you can just merge back from master then everything will be ok